cpp/arcticdb/version/symbol_list.hpp

```diff
@@ -30,7 +30,10 @@
 #include <mutex>
 
 // FUTURE(GitHub #297)
-#define SYMBOL_LIST_RUNTIME_LOG(...) ARCTICDB_RUNTIME_DEBUG(log::version(), "Symbol List: {}: ", __func__, __VA_ARGS__)
```

Comment:
don't see what was wrong with having the prefix?

Reply:
Nothing is wrong with the prefix. `__VA_ARGS__` includes strings with `{}` which aren't effectively replaced so most messages were just blank.

Now `message` is concatenated rather than formatted this will work.